.release/vault-secrets-operator-artifacts.hcl

```
@@ -6,12 +6,17 @@ artifacts {
   zip = [
     "vault-secrets-operator_${version}_linux_amd64.zip",
     "vault-secrets-operator_${version}_linux_arm64.zip",
+    "vault-secrets-operator_${version}_linux_s390x.zip",
   ]
   container = [
     "vault-secrets-operator_release-default_linux_amd64_${version}_${commit_sha}.docker.tar",
     "vault-secrets-operator_release-default_linux_arm64_${version}_${commit_sha}.docker.tar",
+    "vault-secrets-operator_release-default_linux_s390x_${version}_${commit_sha}.docker.tar",
     "vault-secrets-operator_release-ubi-redhat_linux_amd64_${version}_${commit_sha}.docker.redhat.tar",
+    "vault-secrets-operator_release-ubi-redhat_linux_arm64_${version}_${commit_sha}.docker.redhat.tar",
+    "vault-secrets-operator_release-ubi-redhat_linux_s390x_${version}_${commit_sha}.docker.redhat.tar",
```

I think these will be "release-ubi" instead of "release-ubi-redhat" with the extra redhat job removed (it used a slightly different docker target IIRC).

```suggestion
    "vault-secrets-operator_release-ubi_linux_amd64_${version}_${commit_sha}.docker.redhat.tar",
    "vault-secrets-operator_release-ubi_linux_arm64_${version}_${commit_sha}.docker.redhat.tar",
    "vault-secrets-operator_release-ubi_linux_s390x_${version}_${commit_sha}.docker.redhat.tar",
```